templates/docs/examples/patterns/code-snippet/dropdown-multiple.html

```diff
@@ -0,0 +1,72 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code snippet / Multiple dropdowns{% endblock %}
+
+{% block standalone_css %}patterns_code-snippet{% endblock %}
+
+{% block content %}
+<div class="p-code-snippet">
+  <div class="p-code-snippet__header">
+    <h5 class="p-code-snippet__title">Installing Thunderbird as a snap</h5>
+
+    <div class="p-code-snippet__dropdowns">
+      <select class="p-code-snippet__dropdown" name="menu1-select" id="menu1-select">
+        <option value="menu2-select">amd64</option>
+        <option value="menu3-select">i386</option>
+      </select>
+
+      <select class="p-code-snippet__dropdown" name="menu2-select" id="menu2-select">
+        <option value="stable-panel">stable</option>
+        <option value="beta-panel">beta</option>
+        <option value="edge-panel">edge</option>
+      </select>
+
+      <select class="p-code-snippet__dropdown u-hide" name="menu3-select" id="menu3-select">
```

Comment:
If possible I'd like to avoid showing/hiding the selects. Not to complicate the examples too much.

Maybe we could make it work with the same list of options for both architectures?
The real implementation of that in products will need to be quite custom anyway, so I think we should make our example simpler - to focus mostly on showing the HTML structure, not the ways to change the selects with JS.

Reply:
I missed that, why is it hidden in the first place?

Reply:
@bartaz @lyubomir-popov fair enough, it's a bit too convoluted as is - showing/hiding a select was a reasonably simple way to have the left most select determine what options were available in the right most select, which in turn would determine what is shown in the panel below.